specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2023-05-18-preview/examples/deleteController.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "613192d7-503f-477a-9cfe-4efc3ee2bd60",
+    "resourceGroupName": "TestRG",
+    "resourceName": "testcontroller",
+    "api-version": "2023-05-18-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "description": "OK."
+    },
+    "202": {
+      "headers": {},
```

Comment:
Since this is Async Delete, all such long running operations should have location or azure-AsyncOperation in response headers in examples.

See: [https://github.com/Azure/azure-rest-api-specs/pull/24097/checks?check_run_id=13674023663](https://github.com/Azure/azure-rest-api-specs/pull/24097/checks?check_run_id=13674023663).

Reply:
@visingla-ms thanks for pointing this out. I noticed that and since I'm not familiar with the swagger spec review process I was waiting for guidance on this. I've not touched the existing Delete APIs (they are the same as in the previous stable version that our customers are using). Should I fix them or it's ok to keep them as-is ?

Reply:
@visingla-ms also, we don't use the powershell sdk. I was wondering how blocking is that check too.

Reply:
Added LRO Location header to DELETE apis